providers/openebs/installers/operator/0.6/ansible/openebs_cleanup.yaml

```diff
@@ -9,6 +9,28 @@
 
   tasks:
     - block:
+          - block:
+              ## RECORD START of litmus job
+              - name: Record test instance/run ID
+                set_fact:
+                  run_id: "{{ lookup('env','RUN_ID') }}"
+
+              - name: Construct testname appended with runID
+                set_fact:
+                  test_name: "{{ cleanup_test_name }}-{{ run_id }}"
+                when: lookup('env','RUN_ID')
+
+              - name: Generate the litmus result CR to reflect Start of Cleanup

```

Comment:
The Result CR generate and apply - can be outside the block meant for altering test_name upon presence of the RunID ENV